schemas/test_cases/v0/eventually.yaml

```diff
@@ -0,0 +1,60 @@
+- name: azure connection string valid
+  complete_as:
+    - http://determined.ai/schemas/expconf/v0/azure.json
+  case:
+    type: azure
+    container: container_name
+    connection_string: connection_string
+
+- name: azure account url valid
+  complete_as:
+    - http://determined.ai/schemas/expconf/v0/azure.json
+  case:
+    type: azure
+    container: container_name
+    account_url: account_url
+
+- name: azure account url and credential valid
+  complete_as:
+    - http://determined.ai/schemas/expconf/v0/azure.json
+  case:
+    type: azure
+    container: container_name
+    account_url: account_url
+    credential: credential
+
+- name: azure no container or connection info invalid
+  completeness_errors:
+    http://determined.ai/schemas/expconf/v0/azure.json:
+      - "<config>: container is a required property"
+      - "<config>: Exactly one of connection_string or account_url must be set"
+  case:
+    type: azure
+
+- name: azure no container but connection_string invalid
+  completeness_errors:
+    http://determined.ai/schemas/expconf/v0/azure.json:
+      - "<config>: container is a required property"
+  case:
+    type: azure
+    connection_string: connection_string
+
+- name: azure both connection_string and account_url invalid
+  completeness_errors:
+    http://determined.ai/schemas/expconf/v0/azure.json:
+      - "<config>: Exactly one of connection_string or account_url must be set"
+  case:
+    type: azure
+    container: container_name
+    connection_string: connection_string
+    account_url: account_url
+
+- name: azure no container and both connection_string and account_url invalid
+  completeness_errors:
+    http://determined.ai/schemas/expconf/v0/azure.json:
+      - "<config>: container is a required property"
+      - "<config>: Exactly one of connection_string or account_url must be set"
+  case:
+    type: azure
+    connection_string: connection_string
+    account_url: account_url
```

Comment:
small nit: can you configure your editor to save files with a proper line ending on the last file?  Having consistent whitespace treatment helps cut down on unnecessary churn in PRs when other devs edit the same files later, and in this case, "every line has a line ending" is the de-facto standard in unix